### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by Abe1able**
### In this commit, I have

- used media queries to make the desktop version of my portfolio.
- used html and css best practices to produce quality code.
- made sure there are no linter errors in my code.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -430,3 +533,249 @@ form textarea {
...
+    height: 330px;
+    box-shadow: none;
+  }
+
+  .project-detail-article {
+    margin: 70px 120px;
+  }
+}
+
+@media screen and (min-width: 995px) {
```
Comment (vechicin) [inline styles.css]: You should only have one media query and it should be in the range of 768px. Please fix it.

Screen Shot 2022-08-25 at 8 49 56

File: styles.css [hunk 2]
```diff
@@ -33,7 +33,7 @@ button:disabled {
 
 .header {
   position: fixed;
-  width: 90vw;
+  width: 95vw;
   display: flex;
   justify-content: space-between;
   align-items: center;
```
Comment (vechicin) [inline styles.css:39]: Please make sure your design matches the template design.

Screen Shot 2022-08-25 at 8 52 19

Screen Shot 2022-08-25 at 8 52 37

File: styles.css [hunk 3]
```diff
@@ -430,3 +533,249 @@ form textarea {
...
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    flex: 0.5;
+  }
+
+  .about-section-skill {
+    padding-right: 30px;
+    flex: 0.5;
+  }
```
Comment (vechicin) [inline styles.css:665]: Please make sure you match the template design.

Screen Shot 2022-08-25 at 8 54 41

Screen Shot 2022-08-25 at 8 54 56

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Abe1able ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good job matching the template design overall.

## Required Changes ♻️

- [x] You did not use `display: grid` to lay out your elements in the desktop version. Please make sure you do so as it is a project requirement.
Screen Shot 2022-08-25 at 9 00 43

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (Abe1able) [reply styles.css]: Thank you, fixed.

File: styles.css (hunk 2 above)
Comment (Abe1able) [reply styles.css:39]: Thank you, fixed.

**Review by phelian23 — APPROVED** [review]
Hi @Abe1able,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**